.circleci/config.yml

```diff
@@ -36,13 +38,19 @@ jobs:
 
   deploy-to-prod:
     docker:
-      - image: circleci/node:12.6
+      - image: 'cimg/python:3.6'
```

Comment:
What does this change from `node` to `python` do for us?

Reply:
I honestly did not even try with node, but docs for the orb (basically a piece of pluggable functionality) used the python image. I assume the AWS cli depends on it.